openslides_backend/action/list_of_speakers/delete_all_speakers.py

```diff
@@ -0,0 +1,47 @@
+from typing import Iterable
+
+from ...models.models import ListOfSpeakers
+from ...shared.exceptions import ActionException
+from ...shared.patterns import FullQualifiedId
+from ..base import Action, ActionPayload, DataSet, WriteRequestElement
+from ..default_schema import DefaultSchema
+from ..register import register_action
+from ..speaker.create_update_delete import SpeakerDeleteAction
+
+
+@register_action("list_of_speakers.delete_all_speakers")
+class ListOfSpeakersDeleteAllSpeakersAction(Action):
+    """
+    Action to delete all speakers of a list of speakers.
+    """
+
+    model = ListOfSpeakers()
+    schema = DefaultSchema(ListOfSpeakers()).get_default_schema(
+        required_properties=["id"],
+        title="Delete all speakers of list of speakers",
+        description="Action to remove all speakers from the given list of speakers.",
+    )
+
+    def prepare_dataset(self, payload: ActionPayload) -> DataSet:
+        data = []
+        for instance in payload:
+            list_of_speakers = self.fetch_model(
+                FullQualifiedId(self.model.collection, instance["id"]),
+                mapped_fields=["speaker_ids"],
+            )
+            if not list_of_speakers.get("speaker_ids"):
+                raise ActionException(
+                    f"List of speakers {instance['id']} has no speakers."
```

Comment:
Do we really want to throw an exception in this case? I think it's cleaner and more on course with the rest of the actions to just to nothing here. If you want to throw an exception, please add a test case for this.